docs/en/ingest-management/elastic-agent/elastic-agent-providers.asciidoc

```diff
@@ -201,12 +197,10 @@ In this, this option should be carefully considered.
 Provides the option to enable leaderelection between a set of Agents
 running on Kubernetes. Only one Agent at a time will be the holder of the leader
 lock and based on this, configurations can be enabled with the condition
-that the Agent holds the leadership. This can be useful in cases where we want
-only Agent between a set of Agents to collect cluster wide metrics for the
-Kubernetes cluster like from `kube-state-metrics` endpoint.
+that the Agent holds the leadership. This is useful in cases where the agent between a set of agents collects cluster wide metrics for the Kubernetes cluster, such as the `kube-state-metrics` endpoint.
 
-Provider needs a `kubeconfig` file so as to establish connection to Kubernetes API,
-or it can automatically reach the API if it runs in an inCluster environment (Agent runs as Pod).
+Provider needs a `kubeconfig` file to establish a connection to Kubernetes API.
+It can automatically reach the API if it's running in an inCluster environment (Agent runs as Pod).
```

Comment:
I wasn't sure if agent and pod should be capitalized here, similar as called out above.

Reply:
See previous answer